src/qililab/qprogram/structured_program.py

```diff
@@ -99,6 +102,15 @@ def variables(self) -> list[Variable]:
         """
         return list(self._variables)
 
+    @property
+    def get_crosstalk(self) -> CrosstalkMatrix | None:
```

Comment:
If it's a property then `crosstalk` is a better name. `get_crosstalk` implies a method.
```suggestion
    def crosstalk(self) -> CrosstalkMatrix | None:
```